ClassRegistration/ClassRegistration.App/Controllers/EnrollmentController.cs

```diff
@@ -0,0 +1,89 @@
+﻿
+using ClassRegistration.App.Interfaces;
+using ClassRegistration.App.Repositories;
+using ClassRegistration.DataAccess.Entity;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using System.Security.Cryptography.X509Certificates;
+
+// For more information on enabling Web API for empty projects, visit https://go.microsoft.com/fwlink/?LinkID=397860
+
+namespace ClassRegistration.App.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class EnrollmentController : ControllerBase
+    {
+
+        /// <summary>
+        /// private IEnrollment field.
+        /// </summary>
+        private IEnrollmentRepo _enrollmentRepository;
+
+        
+        /// <summary>
+        /// Constructor initializes IEnrollment field.
+        /// </summary>
+        /// <param name="enrollmentRepo"></param>
+        public EnrollmentController(IEnrollmentRepo enrollmentRepo)
+        {
+            this._enrollmentRepository = enrollmentRepo;

```

Comment:
The '_' is added so that we don't need to specify 'this' in setters. Conventionally, we either use the '_' and omit the 'this' keyword, or use the 'this' keyword and omit the '_'

Reply:
Resolved.